content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -28,7 +28,7 @@ When creating SLOs, you can choose from the following types:
 |-----------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
 | **Supported data types**                                              | Metrics with type of count, rate, or distribution                                                                                         | Metric Monitor types, Synthetic Monitors, and Service Checks                                                                                                        | All metric types (including gauge metrics)                                           |
-| **Functionality for SLO with Groups**                                               | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                                                                                  | Supported for SLOs with a single multi alert Monitor<br><br>**Option 1:** SLO calculated based on all groups (can view 5 worst groups in SLO side panel and SLO summary widget)<br>**Option 2:** SLO calculated based on up to 20 selected groups (can view all selected groups in SLO side panel and SLO summary widget)                                                                                                                                | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                           |
+| **Functionality for SLO with Groups**                                               | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                                                                                  | Supported for SLOs with a single multi alert Monitor<br><br>**Option 1:** SLO calculated based on all groups (can view all groups in SLO side panel and SLO summary widget for SLOs with up to 5,000 groups)<br>**Option 2:** SLO calculated based on up to 20 selected groups (can view all selected groups in SLO side panel and SLO summary widget)                                                                                                                                | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                           |
```

Comment:
@estherk15 So if your SLO has 5000 groups or less, you can see all the groups. If it has more than 5,000 groups you can't see any (technical limitations). Lmk if you think there is a better wording option

Reply:
Is this common? Do a lot of customer have SLOs with more than 5,000 groups? If not, recommend adding this limitation to the Support docs instead of public docs.
```suggestion
| **Functionality for SLO with Groups**                                               | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                                                                                  | Supported for SLOs with a single multi alert Monitor<br><br>**Option 1:** SLO calculated based on all groups (can view all groups in SLO side panel and SLO summary widget. Not supported for SLOs with more 5,000 groups)<br>**Option 2:** SLO calculated based on up to 20 selected groups (can view all selected groups in SLO side panel and SLO summary widget)                                                                                                                                | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                           |
```